.circleci/config.yml

```
@@ -316,12 +316,62 @@ jobs:
       - run:
           name: Generate SQL content
           command: |
+            # Check if the generated-sql branch can simply be re-used and SQL generation can be skipped.
+            git clone -b generated-sql git@github.com:mozilla/bigquery-etl ~/generated-sql
```

nit, feel free to ignore but I'd prefix these dirs with `remote-` for a little more clarity 
```suggestion
            git clone -b generated-sql git@github.com:mozilla/bigquery-etl ~/remote-generated-sql
```
similarly below as `remote-bigquery-etl-main` (or just `remote-main`)